### ISSUE TITLE
Tests fail on non-Debian systems

### ISSUE THREAD

**Issue by eserte** [dod38fr/Software-LicenseMoreUtils#1]
On FreeBSD an CentOS6 systems the test suite fails starting with:
```
#   Failed test 'GPL-1 summary'
#   at t/license-summary.t line 17.
#                   ''
#     doesn't match '(?^:can be found in '/usr/share/common-licenses/GPL-1')'

#   Failed test 'GPL-2 summary'
#   at t/license-summary.t line 29.
#                   ''
#     doesn't match '(?^:can be found in '/usr/share/common-licenses/GPL-2)'

#   Failed test 'GPL-1 or later summary'
#   at t/license-summary.t line 50.
#                   ''
#     doesn't match '(?^:any later version)'
# Looks like you failed 3 tests of 8.
t/license-summary.t ...... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/8 subtests 
```
This directory does not exist on these systems --- this seems to be a Debianism.

**Comment by dod38fr** [comment]
Indeed. I've written this module with other distributions in mind. I guess that the tests were not written the same way... oops.

This will fixed soon.

Thanks for the report

**Comment by dod38fr** [comment]
Fixed in v0.002. Thanks for the heads-up.
